v3/vertex/t75/t75.json

```diff
@@ -0,0 +1,16 @@
+{
+    "name": "T75",
+    "vendorId": "0x28E9",
+    "productId": "0x3006",
+    "matrix": {"rows": 6, "cols": 15},    
+    "layouts":{      
+      "keymap":[
+            [{"c": "#777777"},"0,0",{"c": "#cccccc","x":0.5},"0,1","0,2","0,3","0,4",{"x":0.5},"0,5","0,6","0,7","0,8",{"x":0.5},"0,9","0,10","0,11","0,12",{"x":0.5},"0,13"],
+            ["1,0","1,1","1,2","1,3","1,4","1,5","1,6","1,7","1,8","1,9","1,10","1,11","1,12",{"w":2},"1,13","1,14"],
+            [{"w":1.5},"2,0","2,1","2,2","2,3","2,4","2,5","2,6","2,7","2,8","2,9","1,10","2,11","2,12",{"w":1.5},"2,13","2,14"],
+            [{"w":1.75},"3,0","3,1","3,2","3,3","3,4","3,5","3,6","3,7","3,8","3,9","3,10","3,11",{"c": "#777777","w":2.25},"3,12",{"c": "#cccccc"},"3,13"],
+            [{"c": "#aaaaaa", "w":2.25},"4,0",{"c": "#cccccc"},"4,1","4,2","4,3","4,4","4,5","4,6","4,7","4,8","4,9","4,10",{"c": "#aaaaaa", "w":1.75},"4,11",{"c":"#777777"},"4,12",{"c": "#cccccc"},"4,13"],
+            [{"c": "#aaaaaa", "w":1.25},"5,0",{"w":1.25},"5,1",{"w":1.25},"5,2",{"w":6.25}, {"c": "#cccccc"} ,"5,5",{"c": "#aaaaaa","w":1.25},"5,9",{"w":1.25},"5,10",{"c": "#777777","x":0.5},"5,11","5,12","5,13"]
+        ]
+    }
+}
```

Comment:
Just a matter of file formatting to align with the other files.
Once applied the PR can be merged right away.
Thanks.
```suggestion
{
  "name": "T75",
  "vendorId": "0x28E9",
  "productId": "0x3006",
  "matrix": { "rows": 6, "cols": 15 },
  "layouts": {
    "keymap": [
      [
        {
          "c": "#777777"
        },
        "0,0",
        {
          "c": "#cccccc",
          "x": 0.5
        },
        "0,1",
        "0,2",
        "0,3",
        "0,4",
        {
          "x": 0.5
        },
        "0,5",
        "0,6",
        "0,7",
        "0,8",
        {
          "x": 0.5
        },
        "0,9",
        "0,10",
        "0,11",
        "0,12",
        {
          "x": 0.5
        },
        "0,13"
      ],
      [
        "1,0",
        "1,1",
        "1,2",
        "1,3",
        "1,4",
        "1,5",
        "1,6",
        "1,7",
        "1,8",
        "1,9",
        "1,10",
        "1,11",
        "1,12",
        {
          "w": 2
        },
        "1,13",
        "1,14"
      ],
      [
        {
          "w": 1.5
        },
        "2,0",
        "2,1",
        "2,2",
        "2,3",
        "2,4",
        "2,5",
        "2,6",
        "2,7",
        "2,8",
        "2,9",
        "1,10",
        "2,11",
        "2,12",
        {
          "w": 1.5
        },
        "2,13",
        "2,14"
      ],
      [
        {
          "w": 1.75
        },
        "3,0",
        "3,1",
        "3,2",
        "3,3",
        "3,4",
        "3,5",
        "3,6",
        "3,7",
        "3,8",
        "3,9",
        "3,10",
        "3,11",
        {
          "c": "#777777",
          "w": 2.25
        },
        "3,12",
        {
          "c": "#cccccc"
        },
        "3,13"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 2.25
        },
        "4,0",
        {
          "c": "#cccccc"
        },
        "4,1",
        "4,2",
        "4,3",
        "4,4",
        "4,5",
        "4,6",
        "4,7",
        "4,8",
        "4,9",
        "4,10",
        {
          "c": "#aaaaaa",
          "w": 1.75
        },
        "4,11",
        {
          "c": "#777777"
        },
        "4,12",
        {
          "c": "#cccccc"
        },
        "4,13"
      ],
      [
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "5,0",
        {
          "w": 1.25
        },
        "5,1",
        {
          "w": 1.25
        },
        "5,2",
        {
          "w": 6.25
        },
        {
          "c": "#cccccc"
        },
        "5,5",
        {
          "c": "#aaaaaa",
          "w": 1.25
        },
        "5,9",
        {
          "w": 1.25
        },
        "5,10",
        {
          "c": "#777777",
          "x": 0.5
        },
        "5,11",
        "5,12",
        "5,13"
      ]
    ]
  }
}
```